README.md

```diff
@@ -9,6 +11,7 @@ This is the official `Angular in darija` landing page containing all video sessi
 - Run `npm start` 
 - Tests `npm run test`  
 - End to end tests `npm run e2e`
+- Conventional git commit `npm run cz` (please use instead of `git commit` whenever possible)
```

Comment:
If we use husky (see below) we'll enforce using cz on each commit.

Reply:
my bad, there should be a hook for that, will add and republish